### ISSUE TITLE
Unable to CreateElement(VideoPlayer) in nativescript-vue (undefined is not a constructor, evaluating 'new viewClass()')

### ISSUE THREAD

**Issue by starryalley** [nstudio/nativescript-videoplayer#107]
I use nativescript-vue and typescript. Registered like this:

```
import Video from 'nativescript-videoplayer'
Vue.registerElement("VideoPlayer", () => Video);
```

Use like this:

`<VideoPlayer debug=true ref="player" src="https://clips.vorwaerts-gmbh.de/big_buck_bunny.mp4" :autoplay="autoplay" width="100%" />`

When loading it failed at CreateElement:

```
CONSOLE LOG '{NSVue (Vue: 2.5.13 | NSVue: 1.3.1)} -> CreateElement(VideoPlayer)'
***** Fatal JavaScript exception - application has been terminated. *****
....
JavaScript error:
undefined:5821:37: JS ERROR TypeError: undefined is not a constructor (evaluating 'new viewClass()')

```

When digging into the place just before viewClass() is called, the resolver actually returns 'undefined':

`CONSOLE LOG "'entry:[object Object] entry.resolver=function () {\134n    return _nativescriptVideoplayer2.default;\134n} returns:undefined'"`

Anyway to solve this? Not sure if it's a setting issue.

**Comment by starryalley** [comment]
I found an answer but I still don't know why.

I changed the import to require() like this:

`Vue.registerElement("VideoPlayer", () => require("nativescript-videoplayer").Video);`

And the resolver function becomes:
`{\134n    return __webpack_require__(105).Video;\134n}`

And it returns: `function Video() {.....`

And it works. 

Any reason why? Is it a setting issue of my environment? If so I'd like to close this. Thank you.
